core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -144,6 +148,15 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-back-to-collection-container {
+    margin-top: 25px;
```

Comment:
I think it is worth making another collection, actually, just so that you become more familiar with the available functionality. Note that you can reload some of the demo explorations first, and then use those in your collection.

One tip: you can use the `--save_datastore` argument when re-running start.sh in order to save the datastore from the previous run.

Another tip: you can try making your browser narrower. At some point the display should switch to "small-width screen mode".

For your second comment -- I don't know! Basically, make it look as nice as you can, and post screenshots in this thread. Rachel can then give you advice.